src/Controllers/SecurityController.php

```diff
@@ -24,38 +24,40 @@ public function __construct(RendererInterface $renderer, SecurityServices $secur
         $this->securityServices = $securityServices;
     }
 
-    public function logout()
-    {
-        $this->securityServices->logout();
-        $location = 'Location: http://quizApp.com/';
-        $body = Stream::createFromString("");
-
-        return new Response($body, '1.1', '301', $location);
-    }
-
     public function showLogin()
     {
-        return $this->renderer->renderView("login.html", []);
+        return $this->renderer->renderView("login.phtml", ['login' => true]);
     }
 
```

Comment:
// if login successful
    // redirect to success page (whatever that page might be)
//  else 
   // Put error message in session.
   // Redirect to login page.


Dupa in login method/page/action

// if session has error messages 
// put error messages as param on the view.
// remove them from session.

Dupa in template login.phtml
// if there are any error messages -> loop over them and show them.